jest.config.js

```diff
@@ -1,7 +1,10 @@
 module.exports = {
   restoreMocks: true,
   coverageDirectory: 'jest-coverage/',
-  collectCoverageFrom: ['<rootDir>/ui/**/swaps/**'],
+  collectCoverageFrom: [
+    '<rootDir>/ui/**/swaps/**',
+    '<rootDir>/ui/ducks/send/**',
```

Comment:
Thank you for adding more code for the coverage and meeting the minimum coverage threshold! This is getting us closer to having coverage for almost the whole repo!